pkg/wifi/iwl.go

```diff
@@ -157,6 +157,14 @@ func (w *IWLWorker) Connect(stdout, stderr io.Writer, a ...string) error {
 	}
 
 	if err := ioutil.WriteFile("/tmp/wifi.conf", conf, 0444); err != nil {
+		var file, err = os.OpenFile("logOutput.txt", os.O_APPEND|os.O_CREATE|os.O_RDWR, 0755)
+		if err != nil {
+			return err
+		}
+		file.WriteString(time.Now().String() + " ")
+		file.WriteString("/temp/wifi.conf: " + err.Error())
+		file.WriteString("\n")
```

Comment:
Maybe use fmt.Sprintf here.

Reply:
os.O_WRONLY not os.O_RDWR since you will only write it.